conda_lockfiles/dumpers/pixi.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+import sys
+from contextlib import nullcontext
+from typing import TYPE_CHECKING
+
+from conda.base.context import context
+from conda.core.prefix_data import PrefixData
+from ruamel.yaml import YAML
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from conda.models.records import PackageRecord
+
+
+def _record_to_pixi_v6_package(record: PackageRecord) -> dict[str, Any]:
+    package = {
+        "conda": record.url,
+    }
+    # add relevent non-empty fields that rattler_lock includes in v6 lockfiles
+    # https://github.com/conda/rattler/blob/rattler_lock-v0.23.5/crates/rattler_lock/src/parse/models/v6/conda_package_data.rs#L46
+    fields = [
+        # name, version, ... have not been observed in lockfile produced by pixi
```

Comment:
To note in this comment: channel, subdir, name, build and version can be deduced from the package url, hence why it's not included here.